src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -288,6 +289,29 @@ export class OrganizationIntegrations extends AsyncComponent<
     });
   };
 
+  getCategories = (features: IntegrationFeature[]): string[] => {
```

Comment:
@NisanthanNanthakumar Don't forget this card also covers the detailed view so you're going to need the category mapping there as well.

Reply:
@NisanthanNanthakumar doesn't seem like the updated category names are in the detailed views yet. Could you update the PR to have a screenshot of that as well?

Reply:
@scefali updated